test/b2gedmntuples_cfg.py

```diff
@@ -100,6 +102,7 @@
 process.load('Configuration.StandardSequences.MagneticField_38T_cff')
 process.load('Configuration.StandardSequences.Services_cff')
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')
+process.load("RecoEgamma/PhotonIdentification/PhotonIDValueMapProducer_cfi")
```

Comment:
Hello Sal,

1) I ll fix this tomorrow!

2) I agree I should have that there, it is a remnant from my tests

3)I am not sure if I understand your comment here. It is necessary for the
ID implementation

Thanks a lot for your comments/advice,

Savvas

On Thu, Jul 9, 2015 at 5:24 PM, rappoccio <notifications@github.com> wrote:

> In test/b2gedmntuples_cfg.py
> <https://github.com/cmsb2g/B2GAnaFW/pull/11#discussion_r34267182>:
>
> > @@ -100,6 +102,7 @@
> >  process.load('Configuration.StandardSequences.MagneticField_38T_cff')
> >  process.load('Configuration.StandardSequences.Services_cff')
> >  process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')
> > +process.load("RecoEgamma/PhotonIdentification/PhotonIDValueMapProducer_cfi")
>
> Does this part need to have an updated recipe to pull from git or
> something?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cmsb2g/B2GAnaFW/pull/11/files#r34267182>.
>